src/main/webapp/user.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Get the user @param user from the user servlet.
+ * @param {String} user
+ * @return a user object
+ */
+function getUser(user) {
+  console.log('entering get user function/n');
+  fetch('/user?user=' + user)
+    .then(errorHandling).then((response) => response.json())
+    .then((user) => {
+      return user;
+    })
```

Comment:
Same as above. If the intention is for this to query a name instead of an ID (which is unique to each user), you should code this for multiple results.